packages/infrastructure/kube_ingress/main.tf

```diff
@@ -181,6 +181,17 @@ locals {
       "nginx.ingress.kubernetes.io/limit-burst-multiplier" = "3"
       "nginx.ingress.kubernetes.io/limit-whitelist"        = join(", ", [])
     },
+
+    var.connection_timeout_seconds == null ? null : {
+      "nginx.ingress.kubernetes.io/proxy-connect-timeout" = tostring(var.connection_timeout_seconds)
```

Comment:
`nginx.ingress.kubernetes.io/proxy-connect-timeout` should always be fairly low: `5`

There is how long it take to establish a connection (not wait on a response). This should almost always be instant and if it isn't instant it needs to be low so that a new connection attempt can be made as fast as possible.

Reply:
This is currently set to `60`, expected? https://github.com/Panfactum/stack/blob/73898384ca622534c280637dc056a07ab1aeaeb0/packages/infrastructure/kube_ingress_nginx/main.tf#L245